namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-prisoner-cell-allocation-prod/resources/rds-postgresql.tf

```diff
@@ -1,65 +0,0 @@
-/*
- * Make sure that you use the latest version of the module by changing the
- * `ref=` value in the `source` attribute to the latest version listed on the
- * releases page of this repository.
- *
- */
-module "rds" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=6.0.1"
-
-  # VPC configuration
-  vpc_name = var.vpc_name
-
-  # RDS configuration
-  allow_minor_version_upgrade  = true
-  prepare_for_major_upgrade    = true
-  performance_insights_enabled = false
-  db_max_allocated_storage     = "500"
-  # enable_rds_auto_start_stop   = true # Uncomment to turn off your database overnight between 10PM and 6AM UTC / 11PM and 7AM BST.
-  # db_password_rotated_date     = "2023-04-17" # Uncomment to rotate your database password.
-  deletion_protection = true
```

Comment:
You will need to raise a PR to set this value to false first.

After merging the PR, you can raise another PR to delete the file.